src/Relation/Nullary/Reflects.agda

```diff
@@ -52,6 +50,16 @@ invert : ∀ {b} → Reflects A b → if b then A else ¬ A
 invert (ofʸ  a) = a
 invert (ofⁿ ¬a) = ¬a
 
+------------------------------------------------------------------------
+-- recompute
+
+-- Given an irrelevant proof of a reflected type, a proof can
+-- be recomputed and subsequently used in relevant contexts.
+
+recompute : ∀ {b} → Reflects A b → Recomputable A
+recompute (ofʸ  a) _ = a
+recompute (ofⁿ ¬a) a = weak-contradiction a ¬a
+
```

Comment:
Should this be renamed to `reflects⇒recomputable` on the model of `Relation.Unary.Consequences` and `dec⇒recomputable`  in `Relation.Nullary.Decidable.Core` above?

Reply:
I must admit that I rather like 'recompute' for this one. But it's not a strong opinion.

Reply:
Sticking with `recompute` for now...